qiskit_sphinx_theme/furo/base/page.html

```diff
@@ -22,19 +22,19 @@
 {% if theme_announcement -%}
 <div class="announcement">
   <aside class="announcement-content">
-    {% block announcement %} {{ theme_announcement }} {% endblock announcement %}
+    {% block announcement %} {{ theme_announcement }} {% endblock announcement  %}    
```

Comment:
This is a bad change. When opening a PR, you want to keep your change as minimal as possible. Changes like this make the PR more noisy.

It's a good practice before you commit your changes to see what will be in the commit.